charts/Chart.yaml

```diff
@@ -104,32 +104,20 @@ annotations:
     - name: prometheus-config-reloader
       image: quay.io/prometheus-operator/prometheus-config-reloader:v0.81.0
     - name: csi-attacher
-      image: registry.k8s.io/sig-storage/csi-attacher:v4.3.0
-    - name: csi-node-driver-registrar
-      image: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.10.0
+      image: registry.k8s.io/sig-storage/csi-attacher:v4.8.1
```

Comment:
how did these get out of date? doesn't ci prevent this from happening?

Reply:
The develop charts of localpvs changed, because of CSI Upgrade. There is no openebs CI involved until next run.

Reply:
I think this will keep on happening with a floating version. The underneath develop charts of dependents will change and there is nothing to update the umbrella post that unless someone does it manually.

Reply:
Oh right... this is a problem :/